HTTP_HEADER_FORMAT.md

```diff
@@ -0,0 +1,88 @@
+# Trace Context HTTP Header Format
+
+Date: 31/03/2017
```

Comment:
I see why this is here, though it will be maintenance causing.. wonder if there's a badge to auto-include the git timestamp?